### DISCUSSION TITLE
The Daily.dev Social Network

### DISCUSSION THREAD

**Discussion by uditalias** [dailydotdev/daily#229] [Feature Requests]
As I see it, Daily is the ultimate place to get the news and follow the topics you like. Instead of subscribing to a dozen newsletters, the news comes to you with a simple `CMD`+`T`  press.

## 👀 Follow and be followed
It will be great if the Daily dev screen will include the option to follow writers and get updates when they publish a new article or story (see below) - after all, I already have a Daily profile :)

## 📖 Daily stories
Could be nice if we could publish stories about articles we read and share our insights about them or just quickly share our knowledge with other Daily users about any tech topic

![daily_stories](https://user-images.githubusercontent.com/1160107/102363460-1abf1a80-3fbe-11eb-9d2f-55d696e20a99.png)

**Comment by idoshamun** [comment]
Hey @uditalias,

Thanks for such wonderful ideas and even mock-ups 🤯
Our roadmap is currently focused on bringing the experience of daily.dev to the web and increasing the content quality and feed customization.
I believe that once we have the webapp in place it will make more sense to implement social-network-like as you suggested.
I think following others may encourage many people to read and learn more 🤩

**Comment by nimrodkra** [comment]
Oh boy that's an idea with a serious explosiveness potential 😅
